__tests__/index.test.js

```diff
@@ -366,3 +365,22 @@ describe("DELETE:/api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("GET: /api/users", () => {
+  test("200: sends an array of users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.users.length).toBeGreaterThan(0);
+        response.body.users.forEach((user) => {
+          expect(user).toHaveProperty("username");
+          expect(user).toHaveProperty("name");
+          expect(user).toHaveProperty("avatar_url");
```

Comment:
It's ideal to also be checking the type of these properties - consider using a flexible matcher like toMatchObject or objectContaining, alongside expect statements for the type you'd want to make this a better test!